src/FsUnit.NUnit/FsUnitTyped.fs

```diff
@@ -16,48 +15,37 @@ module TopLevelOperators =
         Assert.That(actual, FsUnit.Equality.IsNotEqualTo(expected))
 
     [<DebuggerStepThrough>]
-    let shouldContain (expected: 'a) (actual: 'a seq) =
-        let list = List<_>()
-
-        for a in actual do
-            list.Add a
-
-        Assert.Contains(expected, list)
+    let shouldContain (actual: 'a) (expected: 'a seq) =
+        Assert.Contains(actual, Seq.toArray expected)
```

Comment:
We may be able to user `CollectionAssert.Contains` here

Reply:
Did it for `shouldNotContain` aswell.